src/main/tables/107_default_instant_launches.sql

```diff
@@ -0,0 +1,45 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS default_instant_launches (
+    id UUID UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
+
+    -- Indicates the format version of the instant launch
+    -- JSON. Should help us put together conversions between
+    -- format versions without breaking currently running
+    -- services.
+    version SERIAL,
+
+    -- Here's an example of how the instant_launches field
+    -- is supposed to map info types and glob patterns to 
+    -- instant launches. The kind field tells the caller how
+    -- use the top-level key. The default field tells you
+    -- what the default instant launch should be and the
+    -- compatible list contains instant launches that are
+    -- also acceptable for the kind of pattern but aren't the
+    -- default.
+    --
+    -- {
+    --     "*.py" : {
+    --         "kind" : "pattern",
+    --         "default" : "<instant_launch_uuid>",
+    --         "compatible: [
+    --             "<instant_launch_uuid>",
+    --         ],
+    --     },
+
+    --     "<info_type_uuid>" : {
```

Comment:
I think that the info type name will have to be used for this. The `info_type` table in the DE database is out of date and eventually either needs to be updated or removed.

Reply:
I agree -- the names are usable keys anyway, since that's what we store in AVUs, so probably we can just remove any extra surrogate keys in our database, is my feeling.

Reply:
Fixed the comments.